docs/multicluster/antctl.md

```diff
@@ -22,6 +22,9 @@ Namespace.
 - `antctl mc get resourceexport` (or `get resourceexports`, `get re`) command can print
 all ResourceExports, a specified ResourceExport, ResourceExports in a specified
 Namespace, or ResourceExports in a specific member cluster.
+- `antctl mc get membertoken` (or `get membertokens`) command can print all 
```

Comment:
This command runs only in the leader cluster. We should mention that. How about:

`antctl mc get membertoken` (or `get membertokens`) command prints all member tokens, a specified token, or member tokens in a specified Namespace. The command is supported only on a leader cluster.